applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -152,8 +157,19 @@ - (void)initializeWithPluginVersion:(NSString *)pluginVersion sdkKey:(NSString *
         }
     }
     
+    ALSdkSettings *settings = [[ALSdkSettings alloc] init];
+    settings.termsAndPrivacyPolicyFlowSettings.enabled = self.termsAndPrivacyPolicyFlowEnabledToSet.boolValue;
+    settings.termsAndPrivacyPolicyFlowSettings.privacyPolicyURL = self.privacyPolicyURLToSet;
+    settings.termsAndPrivacyPolicyFlowSettings.termsOfServiceURL = self.termsOfServiceURLToSet;
+    settings.termsAndPrivacyPolicyFlowSettings.debugUserGeography = [self toAppLovinConsentFlowUserGeography: self.debugUserGeographyToSet];
```

Comment:
We should do something similar to what we do on Android. Check for value existence before setting them.